Makefile

```diff
@@ -15,24 +15,32 @@ clean:
 	docker-compose kill
 	docker system prune -f
 
-composer:
+copyJsLib:
+	cp ./node_modules/@simplewebauthn/browser/dist/bundle/index.umd.min.js ./www/simplewebauthn/browser.js
+	cp ./node_modules/@simplewebauthn/browser/LICENSE.md ./www/simplewebauthn/LICENSE.md
+
+deps:
 	docker-compose run --rm composer bash -c "composer install --no-scripts"
+	docker-compose run --rm node npm install --ignore-scripts
+	make copyJsLib
 
-composerupdate:
+depsupdate:
 	docker-compose run --rm composer bash -c "composer update --no-scripts"
+	docker-compose run --rm node npm update --ignore-scripts
```

Comment:
Want one of these?
```
composer show -D --format=json > installed-packages.json
```
Maybe not this time due to the time pressure?

Reply:
Yes to both 🙂